charts/ecosystem/templates/api.yaml

```diff
@@ -100,15 +100,27 @@ spec:
         - file:/bootstrap.properties
         env:
         - name: GALASA_CONFIG_STORE
-          value: etcd:http://{{ .Release.Name }}-etcd:2379
+          value: {{- if contains "RELEASE_NAME" .Values.cpsUrl }}
+            etcd:{{ .Values.cpsUrl | replace "RELEASE_NAME" .Release.Name }}
```

Comment:
Minor suggestion: To reduce some of the duplication where we set the `cpsUrl` value - we could add a named template into the `_helpers.tpl` file for each of the urls? That way we would only need to include the template rather than the whole if-else-end block

Reply:
Thanks Eamonn, I hadn't seen that file but it looks a lot cleaner now with that improvement.